components/feature/readerview/src/main/assets/extensions/readerview/readerview.js

```diff
@@ -191,15 +191,20 @@ class ReaderView {
     const readingTimeMinsSlow = Math.ceil(length / charactersPerMinuteLow);
     const readingTimeMinsFast  = Math.ceil(length / charactersPerMinuteHigh);
 
-    // TODO remove moment.js: https://github.com/mozilla-mobile/android-components/issues/2923
-    // We only want to show minutes and not have it converted to hours.
-    moment.relativeTimeThreshold('m', Number.MAX_SAFE_INTEGER);
-
-    if (readingTimeMinsSlow == readingTimeMinsFast) {
-      return moment.duration(readingTimeMinsFast, 'minutes').locale(lang).humanize();
+    // Construct a localized and "humanized" reading time in minutes. If we
+    // have both a fast and slow reading time we'll show both e.g.
+    // "2 - 4 minutes", otherwise we'll just show "4 minutes".
+    var parts = new Intl.RelativeTimeFormat(lang).formatToParts(readingTimeMinsSlow, 'minute');
+    if (parts.length == 3) {
+      // No need to use part[0] which represent "in".
+      var readingTime = `${parts[1].value} ${parts[2].value}`
```

Comment:
nit: For more clarify, instead of using directly the array's position, we could save the values in variables that give a meaningful names to its content.

Reply:
👍 I can clean this up some more.

Reply:
Done.